automation/repeated_test.sh

```diff
@@ -158,17 +178,31 @@ if should_skip_test_run_due_to_too_many_tests "${NEW_TESTS}"; then
     exit 0
 fi
 
+# for migration tests we need three nodes, thus we need to check whether migration tests are going to be run
+KUBEVIRT_NUM_NODES=2
+if [[ "${NEW_TESTS}" =~ migration ]]; then
+    KUBEVIRT_NUM_NODES=3
```

Comment:
We only need 3 nodes for the tests that include a call to `SkipTestIfNotEnoughNodesWithCPUManager`, not all the migration tests!